CONFIGDOC.md

```diff
@@ -44,21 +45,21 @@ The **Run MATLAB Command** build step enables you to specify MATLAB commands tai
 
 If you specify more than one MATLAB command, use a comma or semicolon to separate the commands. The build fails if the execution of any command results in an error.
 
-![run-matlab-command](https://user-images.githubusercontent.com/48831250/79279466-1d400200-7e7c-11ea-9ef5-0d006f100eea.png)
+![run_matlab_command](https://user-images.githubusercontent.com/48831250/94472499-d8ddbf80-0198-11eb-8f25-ead3c4039f55.png)
+
 
 If you need to specify several MATLAB commands, consider writing a MATLAB script or function as part of your repository and executing this script or function instead. 
 
 Test artifacts are not autogenerated if you choose to run tests using this build step. You can generate your desired test artifacts by configuring the test runner in the script or function that you invoke from the **Command** box.
 
 #### Run MATLAB Tests
-This build step uses a default setting to run tests authored using the MATLAB Unit Testing Framework or Simulink Test&trade;. If your source code is organized into files and folders within a MATLAB project, then the plugin includes any test files in the project that have been labeled as `Test`. If your code does not leverage a MATLAB project or uses a MATLAB release before R2019a, then the plugin includes all tests in the project workspace, including its subfolders. 
+This build step uses a default setting to run tests authored using the MATLAB Unit Testing Framework or Simulink Test&trade;. If your source code is organized into files and folders within a MATLAB project, then the plugin includes any test files in the project that have been labeled as `Test`. If your code does not leverage a MATLAB project or uses a MATLAB release before R2019a, then the plugin includes all tests in the project workspace, including its subfolders.
 
-The **Run MATLAB Tests** build step enables you to generate different types of test artifacts. To publish the test results, you can use these artifacts with other Jenkins plugins. By default, the plugin assigns a name to each selected artifact and stores it in the `matlabTestArtifacts` folder of the project workspace. You can override the default artifact name and location by specifying a path relative to the project folder in the **File path** box. (If you leave the text box empty, the plugin does not generate an artifact.)  
- 
-![run_matlab_tests](https://user-images.githubusercontent.com/48831250/84545463-54406180-accd-11ea-890e-7a6bb9ee051b.png)
+ With the **Run MATLAB Tests** build step, you can optionally specify the location of folders containing source code, relative to the project root folder. When you specify a folder in the **Folder path** box, the plugin adds the specified folder and its subfolders to the top of the MATLAB search path. You can specify more folders by clicking **Add Folder**. To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders. 

```

Comment:
what happens if we leave this blank is kinda puzzle still as this statement `To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders t ` is telling that if i select source it will be considered for coverage generation however it does not tell me if this is mandatory if you want to generate code coverage. (Which may not be the case because by default we consider the current workspace as source)

Reply:
Thank you, Nikhil. I see your point. I changed the sentence to this:

If you specify folders, MATLAB uses only the source code in the specified folders and their subfolders to generate a coverage report.